pkg/controller/grouppermission/grouppermission_controller.go

```diff
@@ -2,12 +2,13 @@ package grouppermission
 
 import (
 	"context"
-	"fmt"
+	"regexp"
 	"strings"
+	"fmt"
 
 	managedv1alpha1 "github.com/openshift/rbac-permissions-operator/pkg/apis/managed/v1alpha1"
+  corev1 "k8s.io/api/core/v1"
```

Comment:
```suggestion
	corev1 "k8s.io/api/core/v1"
```

Not sure what happened on this line, the suggestion adds a tab to stay in line with Go style.